src/components/input-text/input-text.scss

```diff
@@ -0,0 +1,327 @@
+:host {
```

Comment:
There appear to be several spots where logical props can be used in place of things that specify height, width, padding and margins.  Should we update those spots?

Reply:
That should probably go in a separate `refactor` issue/PR